stories/Tooltip.tsx

```diff
@@ -6,14 +6,29 @@ import { POPUP_PLACEMENTS } from '../src/constants';
 
 storiesOf('Molecules|Tooltip', module)
     .addDecorator(withKnobs)
-    .add('Tooltip', () => (
-        <div style={{ position: 'relative', display: 'table', margin: '0 auto' }}>
-            <Tooltip
-                placement={select('Placement', POPUP_PLACEMENTS, 'bottom')}
-                content={text('Tooltip text', 'this is my tooltip text')}
-                alwaysVisible={boolean('Always show tooltip', false)}
-            >
-                hover here to see the tooltip
-            </Tooltip>
-        </div>
-    ));
+    .add(
+        'Tooltip',
+        () => (
+            <div style={{ position: 'relative', display: 'table', margin: '0 auto' }}>
+                <Tooltip
+                    placement={select('Placement', POPUP_PLACEMENTS, 'bottom')}
+                    content={text('Tooltip text', 'this is my tooltip text')}
+                    arrow={boolean('Arrow', true)}
+                >
+                    <div>hover here to see the tooltip</div>
```

Comment:
Does that mean that the children always have to be a node? Can we use plain text?

Reply:
@antipin yes, according to tippy.js documentation, a child of the component should have `React.ReactElement<any>` type